From 43379b750f577fcfaf4cdb312b7b13fad9e4dbef Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Wed, 6 Nov 2019 13:58:02 +0100
Subject: [PATCH 03/79] build: use python3 to run landmines

---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git DEPS DEPS
index 07992a0f9b88..a63f71927551 100644
--- DEPS
+++ DEPS
@@ -4114,7 +4114,7 @@ hooks = [
     'name': 'landmines',
     'pattern': '.',
     'action': [
-        'python',
+        'python3',
         'src/build/landmines.py',
     ],
   },
-- 
2.30.1

