From 2161bef5e3d4002f61558d09503c22dffb457f0d Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sun, 24 May 2020 08:15:38 +0200
Subject: [PATCH 06/73] SUSE chromium-libusb_interrupt_event_handler.patch

---
 services/device/usb/usb_context.cc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git services/device/usb/usb_context.cc services/device/usb/usb_context.cc
index 9378fbe4e094c..b3e1ed5bb2949 100644
--- services/device/usb/usb_context.cc
+++ services/device/usb/usb_context.cc
@@ -61,7 +61,11 @@ void UsbContext::UsbEventHandler::Run() {
 
 void UsbContext::UsbEventHandler::Stop() {
   base::subtle::Release_Store(&running_, 0);
+#ifdef LIBUSB_API_VERSION >= 0x01000105
+  libusb_interrupt_event_handler(context_);
+#else
   libusb_interrupt_handle_event(context_);
+#endif
 }
 
 UsbContext::UsbContext(PlatformUsbContext context) : context_(context) {
-- 
2.32.0

