From 2159b940d904331f2669efcdf764fb585cf4b2bf Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sun, 6 Sep 2020 20:39:54 +0200
Subject: [PATCH 10/79] SUSE chromium-fix-char_traits.patch

---
 base/strings/char_traits.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git base/strings/char_traits.h base/strings/char_traits.h
index b193e216ccca..d42a4b671362 100644
--- base/strings/char_traits.h
+++ base/strings/char_traits.h
@@ -67,9 +67,9 @@ constexpr int CharTraits<char>::compare(const char* s1,
   return __builtin_memcmp(s1, s2, n);
 #else
   for (; n; --n, ++s1, ++s2) {
-    if (*s1 < *s2)
+    if ((unsigned char)*s1 < (unsigned char)*s2)
       return -1;
-    if (*s1 > *s2)
+    if ((unsigned char)*s1 > (unsigned char)*s2)
       return 1;
   }
   return 0;
-- 
2.30.1

